controller/work-item-link-blackbox_test.go

```diff
@@ -185,32 +191,45 @@ func (s *workItemLinkSuite) SetupTest() {
 
 	var err error
 
+	// Create a work item link space
+	createSpacePayload := CreateSpacePayload("test-space", "description")
+	_, space := test.CreateSpaceCreated(s.T(), s.workItemSvc.Context, s.workItemSvc, s.spaceCtrl, createSpacePayload)
+	require.NotNil(s.T(), space)
```

Comment:
Well, if this is done in called function then it isn't need to repeat code. I could change that for the space but I won't do those changes in the other lines